pkg/provider/azure_storageaccount.go

```diff
@@ -607,3 +613,13 @@ func isPrivateEndpointAsExpected(account storage.Account, accountOptions *Accoun
 	}
 	return false
 }
+
+func (az *Cloud) getPrivateDNSZone(ctx context.Context, vnetResourceGroup string) (privatedns.PrivateZone, *retry.Error) {
+	klog.V(2).Infof("Getting private dns zone(%s) in resourceGroup (%s)", PrivateDNSZoneName, vnetResourceGroup)
```

Comment:
no need to print `Getting private dns zone` logs

Reply:
since there is only one line of code, I think it's better cancel this function, and use `az.privatednsclient.Get` directly.